tcl/board/xtensa-kc705-flyswatter2.cfg

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2020-2021 Cadence Design Systems, Inc.
```

Comment:
Add below to the first line of all config files.

` # SPDX-License-Identifier: GPL-2.0-or-later `